server/src/routes/auth.ts

```diff
@@ -0,0 +1,111 @@
+import bcrypt from 'bcryptjs';
+import { Router } from 'express';
+import * as yup from 'yup';
+import { User } from '../entity/User';
+
+const authRouter = Router();
+
+yup.ValidationError;
+
+const registerSchema = yup.object().shape({
+  user_name: yup
```

Comment:
we can solve this later, but ideally frontend would send us json in camelCase, so this would be `userName, userEmail` etc.

But since we dont really have a frontend yet. Lets wait with that until we know exactly the format they will send.